run-tests.pl

```diff
@@ -185,6 +201,36 @@ sub usage
 Getopt::Long::Configure("no_passthrough");
 GetOptions($HS_FACTORY->get_options()) or usage(1);
 
+# Check if both options have been set
+if (length($BLACKLIST_FILE) and length($WHITELIST_FILE)) {
+   print STDERR "Not allowed to set both whitelist and blacklist options.\n";
```

Comment:
the perly way to do this is:

```suggestion
   die "Not allowed to set both whitelist and blacklist options.\n";
```

(and the exit is then redundant)